kostal/README.md

```diff
@@ -17,43 +24,35 @@ Is currently working with the following KOSTA inverter modules:
 The plugin can be configured like this:
 
 <pre>
-[KOSTAL]
+[Kostal-PV]
    class_name = Kostal
    class_path = plugins.kostal
-   ip = 10.10.10.10
+   ip = 192.168.1.21
+   cycle = 5
 #   user = pvserver
 #   passwd = pvwr
-#   cycle = 300
+#   datastucture=html
+# use
+#   datastucture=json
+# for UI-Version >6
 </pre>
 
 This plugin retrieves data from a KOSTAL inverter module of a solar energy
 plant.
 
-The data retrieval is done by establishing a network connection to the 
-inverter module and retrieving the status via a HTTP request.
+The data retrieval is done by establishing a network connection to the
+inverter module and retrieving the status via a HTTP/JSON request.
 
-You need to configure the host (or IP) address of the inverter module. Also
-the user and password attributes (user, passwd) can be overwritten, but
+You need to configure the host (or IP) address of the inverter module.
+If datastucture=html is used the user and password attributes (user, passwd) can be overwritten, but
 defaults to the standard credentials.
 
+If datasructure=json is used, no credentials are requred.
+
 The cycle parameter defines the update interval and defaults to 300 seconds.
 
 ## items.conf
 
-### kostal
-
-This attribute references the information to retrieve by the plugin. The
-following list of information can be specified:
-
-  * power_current: The current power of the solar installation
-  * power_total: The total amount of generated energy
-  * power_day: The amount of generated energy for the current day
-  * status: The textual status of the module (off, feeding, ...)
-  * string1_volt ... string3_volt: The current voltage of string 1, 2, 3
-  * string1_ampere ... string3_ampere: The current ampere of string 1, 2, 3
-  * l1_volt ... l3_volt: The current voltage of L 1, 2, 3
-  * l1_watt ... l3_watt: The current watt of L 1, 2, 3
-
```

Comment:
As mentioned before I would like to see a list of possible values for both the html and json format. If possible we could also use the same keys (e.g. `l1_volt` instead of `dc1_v`). In this case we only have one list of supported values and the datastructure doesn't matter - maybe easier for one having the items configured for an old converter and want to switch to a new converter by just change the `datastructure` setting.